### PR TITLE
[NEXTGEN] Improved AntiVoid

### PR DESCRIPTION
**Pull request by be4dev**
* improved antivoid simulation
* increased speedLimiter value range (for example to slowdown with speed potions)

### CODE REVIEW

**Comment by superblaubeere27** [comment]
For which situation is this good?

**Comment by be4dev** [comment]
> For which situation is this good?

it thought that if in any of SAFE_TICKS_THRESHOLD (10) ticks you were falling (for example when you have jumped) and in the next ticksToVoid (relatively big number) you would be falling, you need to blink. This used to cause a lot of unneeded blinking (obv not very good) when you are any near (like 12-15 blocks) any possible falling source
https://youtu.be/ircX8XgQaec (new one)
https://youtu.be/7XLORGMOIRY (old one)

even tho it isn't ideal, it is much better than it used to be

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAntiVoid.kt [hunk 1]
```diff
@@ -78,22 +78,35 @@ object ModuleAntiVoid : Module("AntiVoid", Category.PLAYER) {
...
+                    // 1 s is enough to stop touching keyboard
+                    if (ticksPassed >= 20) {
+                        simulatedPlayer.input.let {
+                            it.pressingForward = false
+                            it.pressingBack = false
+                            it.pressingLeft = false
+                            it.pressingRight = false
+                            it.jumping = false
+                            it.sneaking = false
+                        }
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAntiVoid.kt]: Use DirectionInput.NONE

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAntiVoid.kt [hunk 2]
```diff
@@ -78,22 +78,35 @@ object ModuleAntiVoid : Module("AntiVoid", Category.PLAYER) {
...
                 ModuleDebug.debugParameter(this, "DistanceToVoid", distanceToVoid)
                 val ticksToVoid = (distanceToVoid * 1.4 / 0.98).toInt()
                 ModuleDebug.debugParameter(this, "TicksToVoid", ticksToVoid)
-
                 // Simulate additional ticks to project further movement.
                 // TODO: Fix considering the player's velocity horizontally
                 //   because FallingPlayer did not work as expected and was not very
                 //   consistent, since even slight rotation changes would cause
                 //   the collision check to fail and return impossible results.
                 repeat(ticksToVoid) {
```
Comment (superblaubeere27) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAntiVoid.kt:96]: use `for (ticksPassed in 1..20)` instead

**Comment by 1zun4** [comment]
I pushed a few commits to [be4dev/LiquidBounce](https://github.com/be4dev/LiquidBounce) but for some reason GitHub is not updating this PR. Please re-open it, thank you!

**Pull request merged**